api/host.go

```diff
@@ -32,7 +32,7 @@ type (
 	// to /host/storage - a bunch of information about the status of storage
 	// management on the host.
 	StorageGET struct {
-		Folders []modules.StorageFolderMetadata `json:"folders"`
+		FolderMetadata []modules.StorageFolderMetadata `json:"foldermetadata"`
```

Comment:
From an API usage perspective it seems clearer to have a call that reads something like `storage.folders` rather than `storage.StorageFolderMetadata` when querying the storage folders' paths/sizes. Just my 2¢.